web-admin/tests/bookmarks.spec.ts

```diff
@@ -0,0 +1,261 @@
+import { expect, type Page } from "@playwright/test";
+import { assertUrlParams } from "@rilldata/web-common/tests/utils/assertUrlParams";
+import { test } from "./setup/base";
+import { interactWithTimeRangeMenu } from "@rilldata/web-common/tests/utils/exploreInteractions";
+
+test.describe("Bookmarks", () => {
+  // TODO: use a separate explore to isolate bookmarks
+  test("Create and verify home bookmark", async ({ page }) => {
```

Comment:
Nit: IMO it'd be easier to read the test cases if:
- "Home bookmark" was a test suite
- "Create home bookmark" was an independent test and the various verifications were discrete tests (run serially)

Similarly, "filters-only bookmark" and "standard bookmark" could be their own test suites.